conceptual/EFCore.PG/modeling/generated-properties.md

```diff
@@ -92,17 +92,17 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
 ## GUID/UUID Generation
 
-By default, for GUID key properties, a random GUID is generated client-side by the EF provider and sent to the database.
+By default, for GUID key properties, a GUID is generated client-side by the EF provider and sent to the database. From version 9.0 and onwards, these GUIDs are sequential (version 7), which are more optimized for database indexes (before version 9.0, these GUIDs were random).
```

Comment:
**Suggestion:** Consider emphasis on the word _key_.

I personally read over this without much realization, until I read the next paragraph.